storage/range.go

```diff
@@ -161,9 +161,9 @@ type RangeManager interface {
 	Clock() *hlc.Clock
 	Engine() engine.Engine
 	DB() *client.KV
-	Allocator() *allocator
+	Allocator() *Allocator
```

Comment:
OK, I had to rename the fields to `_allocator` and `_splitQueue`, but I think this is better.